telegram/_message.py

```diff
@@ -775,6 +795,7 @@ async def reply_text(
         allow_sending_without_reply: ODVInput[bool] = DEFAULT_NONE,
         entities: Union[List["MessageEntity"], Tuple["MessageEntity", ...]] = None,
         protect_content: ODVInput[bool] = DEFAULT_NONE,
+        message_thread_id: int = None,
```

Comment:
If the bot gets a message that `message` that has `message.message_thread_id`, what happens if we do 

```
send_message(
    chat_id=message.chat_id,
    text="hi",
    reply_to_message_id=message.message_id,
)
```

Does that make a reply in the correct topic? Or do we have to do 

```
send_message(
    chat_id=message.chat_id,
    text="hi",
    reply_to_message_id=message.message_id,
    message_thread_id=message.message_thread_id
)
```

? Can you try that out? How does the behavior differ between passing both `message_thread_id` and `reply_to_message_id` and only passing `message_thread_id`?

I'm trying to figure out, if passing `message_thread_id` should be automated through the `quote` argument 🤔  This doesn't have to be done in this PR though, that can be done on the api 6.3 branch also later on …

Reply:
> Does that make a reply in the correct topic?

yes

> if passing message_thread_id should be automated through the quote argument
not needed